enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -196,14 +197,15 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
           // and this very first one does not have a file; i.e. file == null
           // but why do we care? perhaps we're assuming there is a jar file in
           // the classpath but what we really need for this one is a classes directory
+          if (file == null) {
+            throw new EnforcerRuleException(
```

Comment:
Why is this an exception? What did the user do wrong, and what should they do differently?

Reply:
Good point. Expanded exception message to include probable cause.